nominal/core.py

```diff
@@ -230,6 +230,11 @@ def update(
         update_dataclass(self, dataset, fields=self.__dataclass_fields__)
         return self
 
+    def add_csv_to_dataset(self, path: str | Path, timestamp_column: str, timestamp_type: TimestampColumnType) -> None:
```

Comment:
we should add a similar method to `NominalClient` for the initial upload as well